hal/src/main/native/athena/CANAPI.cpp

```diff
@@ -108,12 +109,12 @@ void HAL_WriteCANPacket(HAL_CANHandle handle, const uint8_t* data,
   }
   auto id = CreateCANId(can.get(), apiId);
 
+  std::scoped_lock lock(can->periodicSendsMutex);
   HAL_CAN_SendMessage(id, data, length, HAL_CAN_SEND_PERIOD_NO_REPEAT, status);
 
   if (*status != 0) {
     return;
   }
```

Comment:
Question for @ThadHouse: will netcomm still update its internal state if it returns an error?

Reply:
I am unsure. But I do seem to remember a mention from NI that those functions basically can't fail when any of the periodic flags are set.